diff --git a/node_modules/@vitest/coverage-istanbul/dist/provider.js b/node_modules/@vitest/coverage-istanbul/dist/provider.js
index 62b1a4d..4f888fb 100644
--- a/node_modules/@vitest/coverage-istanbul/dist/provider.js
+++ b/node_modules/@vitest/coverage-istanbul/dist/provider.js
@@ -3699,7 +3699,7 @@ class IstanbulCoverageProvider extends BaseCoverageProvider {
 			// Make sure file is not served from cache so that instrumenter loads up requested file coverage
 			await transform(`${filename}?cache=${cacheKey}&vitest-uncovered-coverage=true`);
 			const lastCoverage = this.instrumenter.lastFileCoverage();
-			coverageMap.addFileCoverage(lastCoverage);
+			if(lastCoverage){if(lastCoverage){coverageMap.addFileCoverage(lastCoverage)};}
 			if (debug.enabled) {
 				clearTimeout(timeout);
 				const diff = performance.now() - start;
